include/oneapi/dpl/internal/random_impl/linear_congruential_engine.h

```diff
@@ -182,27 +180,27 @@ class linear_congruential_engine
     typename ::std::enable_if<(_N == 0) && (_FLAG == false)>::type
     skip_seq(unsigned long long __num_to_skip)
     {
-        for (unsigned long long __i = 0; __i < __num_to_skip; ++__i)
+        for (; __num_to_skip > 0; __num_to_skip--)
```

Comment:
do we want it to be --__num_to_skip as for ++_i? (Just ot be aligned everywhere)

Reply:
Agree - fixed